frontend/packages/knative-plugin/integration-tests/features/serverless/actions-on-knative-service.feature

```diff
@@ -28,19 +44,29 @@ Feature: Perform actions on knative service
             Given user is at the Topology page
              When user right clicks on the knative service "kn-service"
               And user selects "Edit labels" from context menu
-              And user adds the label "app=label" to existing labels list in Edit Labels modal
+              And user adds the label "<cluster_local_label>" to existing labels list in Edit Labels modal
               And user clicks on Save button
-             Then user will see the label "app=label" in "kn-service" service side bar details
+             Then user will see the label "<cluster_local_label>" in "kn-service" service side bar details
+              And the knative service "kn-service" got non public address in side bar details
+
+        Examples:
+                  | cluster_local_label                             |
+                  | networking.knative.dev/visibility=cluster-local |
```

Comment:
Is there any reason, why you need examples here.
Because if we have single set of test data, Examples are not necessary

If you are still looking for it, Please replace the "Scenario" with "Scenario Outline"